pre-registration/pre-registration-application-service/pom.xml

```diff
@@ -139,6 +139,7 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+				<version>2.0.2.RELEASE</version>
```

Comment:
Please also signoff you commits. https://github.com/mosip/pre-registration/pull/847/checks?check_run_id=42272797106